### PR TITLE
Windows users can use the Resolve-DnsName command.

### PR DESCRIPTION
**Pull request by ghost**
### Why:

Windows users aren't required to install BIND to verify DNS records but can also use the Resolve-DnsName PowerShell command. 

Closes: 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

<table role="table">
<thead>
<tr>
<th align="left"><strong>Source</strong></th>
<th align="left"><strong>Preview</strong></th>
<th align="left"><strong>Production</strong></th>
<th align="left"><strong>What Changed</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><a href="https://github.com/github/docs/blob/d890ecc92ae8bb054914f60c49b39194cedfefb9/content%2Fpages%2Fconfiguring-a-custom-domain-for-your-github-pages-site%2Fmanaging-a-custom-domain-for-your-github-pages-site.md"><code class="notranslate">pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md</code></a></td>
<td align="left"><a href="https://docs-32449-aa33fe.preview.ghdocs.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site" rel="nofollow">fpt</a><br><a href="https://docs-32449-aa33fe.preview.ghdocs.com/enterprise-cloud@latest/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site" rel="nofollow">ghec</a><br></td>
<td align="left"><a href="https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site">fpt</a><br><a href="https://docs.github.com/enterprise-cloud@latest/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site">ghec</a><br></td>
<td align="left"><a href="https://docs-32449-aa33fe.preview.ghdocs.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#about-custom-domain-configuration">About custom domain configuration</a></td>
</tr>
</tbody>
</table>

image

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).
Source	Preview	Production	What Changed
[pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md](https://github.com/github/docs/blob/3d1d7dac03020699590da8066ffd46f64d226235/content%2Fpages%2Fconfiguring-a-custom-domain-for-your-github-pages-site%2Fmanaging-a-custom-domain-for-your-github-pages-site.md)	[fpt](https://docs-32449-aa33fe.preview.ghdocs.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)
[ghec](https://docs-32449-aa33fe.preview.ghdocs.com/enterprise-cloud@latest/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)
[fpt](https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)
[ghec](https://docs.github.com/enterprise-cloud@latest/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@j0eybrinkman Thanks for opening a PR! ✨ It looks like this is still a draft - could you ping me whenever you're ready? I'll then have the PR reviewed 💛

**Comment by j0eybrinkman** [comment]
@nguyenalex836 Hi Alex, it is ready for review. Thank you!

**Comment by nguyenalex836** [comment]
@j0eybrinkman Thank you for letting me know! I'll get this triaged for review 💛

File: content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Make sure you add your custom domain to your {% data variables.product.prodname_
 
 {% windows %}
 
-The `dig` command, which can be used to verify correct configuration of DNS records, is not included in Windows. Before you can verify that your DNS records are configured correctly, you must install [BIND](https://www.isc.org/bind/).
+The `dig` command, which can be used to verify correct configuration of DNS records, is not included in Windows. Windows users can use the `Resolve-DnsName` PowerShell command or install [BIND](https://www.isc.org/bind/) to verify DNS records are configured correctly.
```
Comment (sophietheking) [inline content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md:33]:
```suggestion
The `dig` command, which can be used to verify correct configuration of DNS records, is not included in Windows. To verify that your DNS records are configured correctly, you can use the `Resolve-DnsName` PowerShell command or install [BIND](https://www.isc.org/bind/).
```

**Review by sophietheking — DISMISSED** [review]
@j0eybrinkman Thanks for opening this PR and contributing to our Docs, we really appreciate it! 💖 I've edited the sentence slightly for clarity, but otherwise it looks good to me, so I'll go ahead and get this merged. 🚀

**Review by sophietheking — APPROVED** [review]

**Pull request merged**